lib/cs_guide_web/templates/page/index.html.eex

```diff
@@ -7,14 +7,17 @@
       <div class="tc mt3">
         <%= form_for @conn, search_all_path(@conn, :index), [method: :get], fn f -> %>
           <div class="flex-ns">
-            <div class="db dib-ns w-100 w-80-ns br2 ba b--cs-light-gray br--left-ns relative">
-              <%= search_input f, :term, placeholder: "Search drinks and venues", class: "h-100 w-100 bn pl3 pv2" %>
-              <div id="use-location" class="f6 lh6 cs-black absolute-vertical-center right-1 pointer dn db-ns">Use my location</div>
-              <div id="use-location-mobile" class="f6 lh6 cs-black absolute-vertical-center right-1 pointer dn-ns relative w1 h1">
-                <img src="/images/location.png" alt="location image" class="absolute-vertical-center right-1">
+
+            <div class="flex w-100 items-center br2 ba b--cs-light-gray br--left-ns bg-white">
+              <%= search_input f, :term, placeholder: "Search drinks and venues", class: "h-100 w-100 bn pl3 pv2 outline-0" %>
+
+              <div id="use-location" class="f6 lh6 cs-black pointer dn db-ns center w4-5">Use my location</div>
+              <div id="use-location-mobile" class="f6 lh6 cs-black pointer dn-ns w1 h1 mh3">
+                <img src="/images/location.png" alt="location image">
               </div>
             </div>
-            <%= submit "Search", class: "dn dib-ns bg-cs-pink white f5 lh5 ph3 pv2 w-20 br2 ba b--cs-pink br--right pointer" %>
+
+            <%= submit "Search", class: "dn dib-ns bg-cs-pink white f5 lh5 ph3 ph2-m pv2 w-20 br2 ba b--cs-pink br--right" %>
```

Comment:
I would keep the "pointer" class on the submit button, no?

Reply:
@SimonLab 🤦‍♀️ Sorry about that. Must have done it in error but good catch. @nelsonic thanks for resolving this 👍